.github/workflows/safer-golangci-lint.yml

```diff
@@ -17,9 +17,9 @@ on:
 
 env:
   GO_VERSION: '1.20'
-  GOLINTERS_VERSION: 1.53.3
+  GOLINTERS_VERSION: 1.54.2
```

Comment:
Why not bump to 1.56.2?

Reply:
> Why not bump to 1.56.2?

I was already using 1.54.2 locally and it passed all lints including go-critic.  Maybe newer version next weekend.